cylc/flow/network/client.py

```diff
@@ -103,17 +109,17 @@ def __init__(
     ):
         super().__init__(zmq.REQ, context=context)
         self.suite = suite
-        host, port, _ = get_location(suite)
+        self.host, self.port, _ = get_location(suite)
```

Comment:
Note, I did this, so that in the `cylc ping` command, we don't have to call `get_location` twice, once when creating the `SuiteRuntimeClient`, and later again to retrieve the `host` and `port` for logging.